settings.py

```diff
@@ -284,7 +284,7 @@
 
 GUNICORN_PID_PATH = "/var/run/gunicorn.pid"
 
-LOG_LEVEL = logging.INFO
+LOG_LEVEL = os.getenv("LOG_LEVEL", logging.INFO)
```

Comment:
Is it a problem if LOG_LEVEL is a string?

Reply:
i set this to always be an int now.